pkg/handler/configs.go

```diff
@@ -632,13 +632,13 @@ func (h *Handler) RemoveRepostReactionConfig(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, response.CreateResponse[any](nil, nil, errors.New("guild_id is required"), nil))
 		return
 	}
-	if req.Emoji == "" {
+	if req.Emoji == "" || (req.Emoji == "" && req.EmojiStop == "") {
```

Comment:
split this into another API to distinguish `message` and `conversation`, you can rename API if its name is meaningless
- `repostReactionGroup.POST("/start-stop", h.CreateConfigRepostReactionStartStop)` (existed)
- `repostReactionGroup.DELETE("/start-stop", h.CreateConfigRepostReactionStartStop)` (create new one)